### PR TITLE
Merge Branch-UG-vinod with Master 

### LINKED ISSUE #30: Add create, remove and display features for events class in User Guide

### PR DESCRIPTION
**Pull request by Vinodjayakumar124**
Closes #30

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -180,6 +182,41 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Creating an event: `create`
+
+Creates a new event in the address book.
+
+Format: `create t/EVENT_TITLE d/DATE t/TIME p/PURPOSE`
+
+* The compulsory parameters are: event title (t), date (d), time (t) and purpose (p)
```
Comment (Benjamin-Sim) [inline docs/UserGuide.md]: Perhaps you can format the command prefixes as inline code to signal to the reader that they are part of the command?

```suggestion
* The compulsory parameters are: event title (`t`), date (`d`), time (`t`) and purpose (`p`)
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -180,6 +182,41 @@ Examples:
...
+* `create t/Banana Discount 10% d/20-04-2022 t/14:00 p/10% discount on all bananas`
+
+### Removing an event: `remove`
+
+Removes an existing event in the address book.
+
+Format: `remove INDEX`
+
+* Removes the event at the specified INDEX. 
+* The index refers to the index number shown in the displayed person list
```
Comment (Benjamin-Sim) [inline docs/UserGuide.md]: I think you are referring to the event list here, not person list.

```suggestion
* The index refers to the index number shown in the displayed event list 
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -180,6 +182,41 @@ Examples:
...
+* `create t/Shoe Sale 30% d/30-05-2022 t/11:00 p/Discount on all shoes for up to 30%`
+* `create t/Banana Discount 10% d/20-04-2022 t/14:00 p/10% discount on all bananas`
+
+### Removing an event: `remove`
+
+Removes an existing event in the address book.
+
+Format: `remove INDEX`
+
+* Removes the event at the specified INDEX.
```
Comment (Benjamin-Sim) [inline docs/UserGuide.md]: Perhaps format INDEX as inline code as well?

```suggestion
* Removes the event at the specified `INDEX`. 
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -180,6 +182,41 @@ Examples:
...
+
+Format: `remove INDEX`
+
+* Removes the event at the specified INDEX. 
+* The index refers to the index number shown in the displayed person list 
+* The index must be a positive integer 1, 2, 3, …, and it must be within the range of the event list index.
+
+Examples:
+* `delete 2 after displaying all events with display removes event at index 2`
+* `delete 7 after displaying all events with display removes event at index 7`
```
Comment (Benjamin-Sim) [inline docs/UserGuide.md]: I think it's better that only the actual command be formatted as inline code; the description should be in plain text.
```suggestion
* `delete 2` after displaying all events with display removes event at index 2
* `delete 7` after displaying all events with display removes event at index 7
```

**Review by Benjamin-Sim — COMMENTED** [review]
All good except for some minor formatting issues! 👍

File: docs/UserGuide.md (hunk 1 above)
Comment (Vinodjayakumar124) [reply docs/UserGuide.md]: Yes, will do this

File: docs/UserGuide.md (hunk 2 above)
Comment (Vinodjayakumar124) [reply docs/UserGuide.md]: Have made this change!

File: docs/UserGuide.md (hunk 3 above)
Comment (Vinodjayakumar124) [reply docs/UserGuide.md]: Have edited this part!

File: docs/UserGuide.md (hunk 4 above)
Comment (Vinodjayakumar124) [reply docs/UserGuide.md]: Yes, thanks! Shouldn't have formatted the whole line

**Review by Benjamin-Sim — APPROVED** [review]
LGTM! 👍

**Pull request merged**